CRVResponse/src/MakeCrvWaveforms.cc

```diff
@@ -56,6 +58,7 @@ void MakeCrvWaveforms::MakeWaveform(const std::vector<double> &times,
   waveform.clear();
 
   if(times.size()==0) return;
+  waveform.resize(8);
```

Comment:
is 8 a hardware or firmware number?  Can it ever change?  If so it should be a configuration parameter.  If not, consider making waveform an std::array<double,8>

Reply:
This waveform vector is not the final waveform. It is the full waveform from which relevant pieces will later be extracted to form the final waveforms. The final waveforms are stored in the CrvDigis as std::array<int16_t,8>.
The full waveform vector has a variable length depending on the distribution of the charges and is being constructed by summing up single PE waveforms. In order to avoid resizing this vector too often, I started the process with a reasonable length of 8. However, I have already made a change that replaces the number 8 with an estimate of the expected vector length.